device/bgp/bgpexcommunity.yaml

```diff
@@ -0,0 +1,55 @@
+components:
+  schemas:
+    Bgp.ExCommunity:
+      description: >-
+        The Extended Communities Attribute is a transitive optional BGP
+        attribute, with the Type Code 16. Community and Extended Communities 
+        attributes to trigger routing decisions, such as acceptance, rejection, 
+        preference, or redistribution. An extended community is an 8-Bytes value.
+        It is also divided into two main parts. The first 2 Bytes of the community
+        encode a type and sub-type fields and the last 6 Bytes carry a unique set of data in a 
+        format defined by the type and sub-type field. Extended communities provide a larger 
+        range for grouping or categorizing communities.
+      type: object
+      properties:
+        type:
+          description: |-
+            Extended Community Type field of 1 Byte.
+             - administrator_as_2octet: Two-Octet AS Specific Extended Community (RFC 4360).
+             - administrator_ipv4_address: IPv4 Address Specific Extended Community (RFC 4360).
+             - administrator_as_4octet:  4-Octet AS Specific Extended Community (RFC 5668).
+             - opaque: Opaque Extended Community (RFC 7432).
+             - evpn: EVPN Extended Community (RFC 7153).  
+             - administrator_as_2octet_link_bandwidth : Link Bandwidth Extended Community (RFC 7153).
+          type: string
+          enum:
+          - administrator_as_2octet
+          - administrator_ipv4_address
+          - administrator_as_4octet
+          - opaque
+          - evpn
+          - administrator_as_2octet_link_bandwidth
+        subtype:
+          description: |-
+            Extended Community Sub Type field of 1 Byte.
+            - route_target: Route Target.
+            - origin: Origin.
+            - extended_bandwidth: Specifies the link bandwidth.
+            - color: Specifies the color value.
+            - encapsulation: Specifies the Encapsulation Extended Community.
+            - mac_address: Specifies the Extended community MAC address.
+          type: string
+          enum:
```

Comment:
Can we have a provision here that user can enter any hex value here apart from these few types/subtypes?

Reply:
We add customer field later too.